cmd/authelia-scripts/main.go

```diff
@@ -42,7 +42,7 @@ var Commands = []AutheliaCommandDefinition{
 	AutheliaCommandDefinition{
 		Name:        "docker",
 		Short:       "Commands related to building and publishing docker image",
-		SubCommands: CobraCommands{DockerBuildCmd, DockerPushCmd},
+		SubCommands: CobraCommands{DockerBuildCmd, DockerPushCmd, DockerManifestCmd},
```

Comment:
As said previously, this should probably not be a specific command but be included in the push command instead.

Reply:
confirmed.